timesketch/lib/emojis.py

```diff
@@ -25,18 +25,46 @@
 
 
 EMOJI_MAP = {
+    '8BALL': emoji('&#1F3B1', 'N/A'),
```

Comment:
why are we adding all these emojis in here if we don't have a purpose for them? that is they are all marked as N/A?

can we somehow find purpose for those?